crypto/dilithium/ml_dsa.h

```diff
@@ -0,0 +1,50 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0 OR ISC
+
+#ifndef SIG_DILITHIUM_H
+#define SIG_DILITHIUM_H
+
+#include <stddef.h>
+#include <stdint.h>
+#include <openssl/base.h>
+#include <openssl/evp.h>
+
+#define MLDSA65_PUBLIC_KEY_BYTES 1952
+#define MLDSA65_PRIVATE_KEY_BYTES 4032
+#define MLDSA65_SIGNATURE_BYTES 3309
+#define MLDSA65_KEYGEN_SEED_BYTES 32
+#define MLDSA65_SIGNATURE_SEED_BYTES 32
+
+// ml_dsa_65_keypair generates an ML-DSA-65 keypair and assigns a public key to
+// |public_key| and a private key to |secret_key|. It returns 0 upon success.
```

Comment:
it can return `-1` when `prefix` string too long, added comment to indicate this [bcbb832](https://github.com/aws/aws-lc/pull/1963/commits/bcbb8322f367532268ad01c5a940a5518b349562)